projects/igniteui-angular/src/lib/grids/filtering/excel-style/excel-style-moving.component.ts

```diff
@@ -15,7 +15,8 @@ export class IgxExcelStyleMovingComponent {
     /**
      * @hidden @internal
      */
-    @HostBinding('class') class = 'igx-excel-filter__move';
+    @HostBinding('class')
+    public class = 'igx-excel-filter__move';
```

Comment:
IMO, can we bind directly to the specific style class instead of the 'wildcard' class attribute?
```typescript
@HostBinding('class.igx-excel-filter__move')
public defaultClass = true;
```

Reply:
Done!